src/Google.Datastore.V1Beta3/DatastoreQueryResults.cs

```diff
@@ -74,22 +68,13 @@ public DatastoreQueryResults(IEnumerable<RunQueryResponse> responses)
             });
 
         /// <summary>
-        /// Returns the results of this query as a sequence of <see cref="RunQueryResponse"/> values
-        /// exactly as returned by the Datastore API. This method is only useful if the application
-        /// wishes to examine the <see cref="RunQueryResponse.Query"/> property; otherwise, use
-        /// <see cref="AsBatches"/> to obtain the sequence of batches.
+        /// This method is for advanced use cases only, where more diagnostic information is required;
```

Comment:
I don't want to be quite that restrictive though. Perhaps "niche use cases" would be better - ego or not, I'm sure you wouldn't argue that "displaying a page of results in a web page" is a niche use case. I'll have another look at this when I'm back from vacation, and see if any alternative wording jumps out... I'd rather have diagnostics as an example of the niche use cases than anything else, to be honest.